modules/commandModule/commandFns.py

```diff
@@ -31,4 +32,30 @@ def write_pigo(newPigo):
 	return
 #if nothing has changed, return the dictionary that was passed in
     else:
-        return
\ No newline at end of file
+        return
+
+def read_pogi() -> dict:
+
+    """
+    Reads all data currently in the POGI file and returns as a dict
+
+    Returns
+    -------
+    dict:
+        contains pogiData as specified in commandModule
+    """
+
+    command = CommandModule(pigoFileDirectory=PIGO_DIR, pogiFileDirectory=POGI_DIR)
+
+    # store all current POGI data into a dict
+    # note: the keys included in the dict are those which are currently valid and working in commandModule
+    #       (some fields are defined in the wiki but are currently commented out)
+    pogiData =  { 'currentAltitude' : command.get_current_altitude(),
```

Comment:
Use standard formatting for dictionaries, there should be a newline after the opening bracket, similar to what is done in write_pogi(). Unfortunately the github editor won't let me give you an example. 
`